package.json

```diff
@@ -4,13 +4,16 @@
   "version": "5.25.0",
   "description": "Utilities to help when developing terra modules.",
   "engines": {
-    "node": "10 || 12|| 14",
-    "npm": "6 || 9"
```

Comment:
Are we okay going ahead and removing npm6 support based on discussion during retro?

Reply:
We will be, because the upgrades won't affect consumers. @nikhitasharma